documentation/release_notes.markdown

```diff
@@ -37,6 +38,14 @@ canonical: "/puppetdb/latest/release_notes.html"
   expected, and is due to internal changes in the way we check if
   catalogs are up-to-date.
 
+### Downgrading
+
+* If you attempt to downgrade to a previous version of PuppetDB, in
+  addition to wiping the database, you will need to clear the queue
+  directory as described [here][kahadb_corruption].  If you see an
```

Comment:
minor note, but I find it more useful to see something like "described in the [documentation on kahadb corruption][kahadb_corruption]" rather than just "here", so that I have an idea of where I'm going. Not a standard we adhere to in general.